subsys/bluetooth/host/audio/endpoint.c

```diff
@@ -404,7 +404,7 @@ static void ep_qos(struct bt_audio_ep *ep, struct net_buf_simple *buf)
 
 	BT_DBG("dir 0x%02x cig 0x%02x cis 0x%02x codec 0x%02x interval %u "
 	       "framing 0x%02x phy 0x%02x rtn %u latency %u pd %u",
-	       ep->chan->cap->type, ep->cis, ep->cis, ep->chan->codec->id,
+	       ep->chan->cap->type, ep->cis_id, ep->cis_id, ep->chan->codec->id,
```

Comment:
I think there is a bug here (unrelated to the PR).
The first `cis_id` should be a `cig_id`

Should be fixed separately, either as an additional commit, or as a separate PR.

Reply:
Since it is just a small bug (wrong value in a debug statement), I just updated it as part of this commit, since this commit changes the value anyhow.